### PR TITLE
PAN-15925: Add response errors to APIError message

### PR DESCRIPTION
**Pull request by MichaelCombs28**

### CODE REVIEW

File: packages/pangea-node-sdk/tsconfig.json [hunk 1]
```diff
@@ -19,6 +19,9 @@
     "noUncheckedIndexedAccess": true,
     "noEmitOnError": true,
     "useDefineForClassFields": true,
+    "target": "es5",
```
Comment (kenany) [inline packages/pangea-node-sdk/tsconfig.json:22]: Why?

**Review by pangea-andrest — DISMISSED** [review]

File: packages/pangea-node-sdk/tsconfig.json [hunk 2]
```diff
@@ -19,6 +19,9 @@
...
+    "lib": ["es2015", "dom"],
```
Comment (kenany) [inline packages/pangea-node-sdk/tsconfig.json:23]: We shouldn't specify `"dom"` since this is a Node.js library.

**Review by kenany — CHANGES_REQUESTED** [review]

File: packages/pangea-node-sdk/tsconfig.json (hunk 1 above)
Comment (MichaelCombs28) [reply packages/pangea-node-sdk/tsconfig.json:22]: Without a target, we get a lot of compiler warnings. ES5 works both in browser and node for the most part.

File: packages/pangea-node-sdk/tsconfig.json (hunk 2 above)
Comment (MichaelCombs28) [reply packages/pangea-node-sdk/tsconfig.json:23]: It should be usable in both, don't know why we have a vanilla-js library when typescript is more common now on the frontend.

**Comment by kenany** [comment]
I've opened #1339 for the additions that tshy is making, plus updating it to v2.0.1 so that it doesn't add `"source"`.

File: packages/pangea-node-sdk/tsconfig.json (hunk 2 above)
Comment (MichaelCombs28) [reply packages/pangea-node-sdk/tsconfig.json]: Removed

File: packages/pangea-node-sdk/tsconfig.json (hunk 1 above)
Comment (MichaelCombs28) [reply packages/pangea-node-sdk/tsconfig.json]: Removed

File: packages/pangea-node-sdk/src/request.ts [hunk 3]
```diff
@@ -377,6 +377,7 @@ class PangeaRequest {
     const fetchOptions: RequestInit = {
       duplex: "half",
       method: options.method,
+      // @ts-ignore
```
Comment (kenany) [inline packages/pangea-node-sdk/src/request.ts:380]: Unnecessary.

File: packages/pangea-node-sdk/package.json
Comment (kenany) [inline packages/pangea-node-sdk/package.json:1]: Pull in 8873574 and then these changes should go away.

**Review by kenany — APPROVED** [review]

**Pull request merged**